### PR TITLE
2019/kw14/fixed_windows_build_docu_fixed_server_start_text

### PR DESCRIPTION
**Pull request by ulfgebhardt**
Fixed windows build, updated readme (prerequisites & requirements), fixed backend server start log output to reflect proper values

Wrong use of `$npm_config` - see here:
> https://docs.npmjs.com/files/package.json#config
> https://docs.npmjs.com/misc/config
> https://docs.npmjs.com/misc/config#per-package-config-settings
> https://stackoverflow.com/questions/34027400/npm-scripts-config-variables-and-command-substitution-not-working-in-package-j

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
...
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extratced Neo4j-Server
+
+Start Neo4j
+```
+neo4j\bin\neo4j start
+```
+and confirm it's running [here](http://localhost:7687)
```
Comment (roschaefer) [inline backend/README.md]:
```suggestion
and confirm it's running [here](http://localhost:7474)
```

File: backend/README.md [hunk 2]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
+
+Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extratced Neo4j-Server
```
Comment (roschaefer) [inline backend/README.md]:
```suggestion
Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
```

File: backend/README.md [hunk 3]
```diff
@@ -63,6 +70,17 @@ docker-compose down -v
 
 Install dependencies:
 
+Download [Neo4j Enterprise Edition](https://neo4j.com/download-center/#releases) and unpack the files.
```
Comment (roschaefer) [inline backend/README.md]:
```suggestion
Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
```

**Comment by ulfgebhardt** [comment]
> Here are some changes I would like to see, could you also update https://github.com/Human-Connection/Nitro-Documentation ?

Please merge it into the repo first

**Comment by roschaefer** [comment]
@ulfgebhardt I restarted the build. If it passes, feel free to hit the merge button.

**Pull request merged**